spec/core/transaction/schemas/EmbeddedTransactionInfoDTO.yml

```diff
@@ -1,8 +1,12 @@
 type: object
 required:
+  - id
   - meta
   - transaction
 properties:
+  id:
+    type: string
+    description: Internal resource identifier.
```

Comment:
Database record id ?

Reply:
@Vektrat , could you confirm if embedded transaction id are / are going to be outside metadata?

Reply:
they are yes, no id should be inside meta